Week-1/Exercise-1.2/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css2?family=Lato' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="site.css">
+    <link rel="stylesheet"
+        href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+</head>
+<body>
+    <nav>
+        <i class="las la-bars"></i>
+        <i class="las la-home"></i>
+        <i class="las la-search"></i>
+        <i class="las la-volume-up"></i>
+        <i class="las la-user"></i>
+        <i class="las la-cog"></i>
+        <i class="lab la-spotify"></i>
+        <i class="lab la-soundcloud"></i>
+    </nav>
+
+    <header>
+        <img id="albumImage" src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg" alt="cattyboard">
+        <div class="header-section">
+            <h1>CattyBoard Top 100 Single Charts (11.06.36)</h1>
+            <p>Unknown Artist</p>
+            <p>2016. Charts. 100 songs</p>
+            <i class="las la-play"><span>Play all</span></i>
+            <i class="las la-plus"><span>Add to</span></i>
+            <i class="las la-ellipsis-h"><span>More</span></i>
+        </div>
+    </header>
+
+    <main class="main-wrapper">
+        <div class="table-container">
+            <div class="table-row">
+                <div class="table-cell">1. One Dance</div>
```

Comment:
@AakashPandya can you use ul..li in place of divs (semantic tags)

Reply:
At parent level, I've fixed by ul...li
Not sure if you're talking about cells...